File: pset6/mario/mario.py
from cs50 import get_int

def getPositiveInt():
    n = get_int("Height:")
    if n < 1 or n > 8:
        n =getPositiveInt()
    return(n)


h = getPositiveInt()
for i in range(1, h+1):
    print(' '*(h-i) + '#'*i + '  ' + '#'*i)

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;
    //asks user for heith value
    while ((height < 1) || (height > 8))
    {
        height = get_int("Heigth:");
    }

    //iterate trough heigth printing the lines
    for (int i = 0; i < height; i++)
    {

        for (int j = height - i; j > 1; j--)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("  ");

        for (int j = i; j >= 0; j--)
        {
            printf("#");
        }

        //create a new line
        printf("\n");


    }

}